### PR TITLE
feature: add swiper for changing hero components

### PR DESCRIPTION
**Pull request by ErickSharp**
## Summary of changes
Used swiperjs to be able to have several project heroes on the front page that you can swipe between. Renamed main header to "A320Header" and added a temporary "A380Header" that has the same elements.

## Screenshots(if applicable)
image

image

image

TODO: Draft how we will communicate the presence of other slides such as radio buttons at the bottom or an arrow on the right/left side of the screen.

### CODE REVIEW

**Comment by NathanInnes** [comment]
Radio buttons should be implemented with this PR and the breaking issue should be fixed instead of being thrown in to worry about later

**Comment by ErickSharp** [comment]
> Radio buttons should be implemented with this PR and the breaking issue should be fixed instead of being thrown in to worry about later

I'll get to work on the radio buttons, let me know if you are able to figure out how to fix the breaking issue.

**Comment by ErickSharp** [comment]
Ok so I put in the little buttons at the bottom and styled them to show whatever slide the user is on. Furthermore, as a workaround for the page breaking caused by scrollintoview, at the moment I am just taking the current window height, and when you click on the "Installer" button, it scrolls down by the height divided by 1.5 which I found to work pretty well. Let me know if any of you have ideas to implement a better fix.

**Comment by NathanInnes** [comment]
Please resolve conflicts @ErickSharp

**Comment by ErickSharp** [comment]
> Please resolve conflicts @ErickSharp

I can't; Header.tsx no longer exists and Installer.tsx looks like normal.

**Comment by ErickSharp** [comment]
> Please resolve conflicts @ErickSharp

No conflicts

File: src/components/home/Installer.tsx [hunk 1]
```diff
@@ -6,16 +6,15 @@ import { Button } from '../Button';
 import InstallerImage from '../../assets/img/InstallerScreenshot.png';
 
 export const Installer: React.FC = () => (
-    <section className="relative w-screen flex flex-col 3xl:flex-row 3xl:justify-center items-center py-6 overflow-hidden bg-blue-dark">
-        <div className="mt-10 max-w-2xl flex flex-col items-center 3xl:items-start text-center 3xl:text-left 3xl:mb-20 3xl:mr-12 space-y-4">
+    <section className="relative h-144 w-screen flex flex-col xl:flex-row items-center overflow-hidden bg-blue-dark" id="installer">
```
Comment (NathanInnes) [inline src/components/home/Installer.tsx:9]:
```suggestion
    <section id="installer" className="relative h-144 w-screen flex flex-col xl:flex-row items-center overflow-hidden bg-blue-dark">
```
Change order

**Comment by NathanInnes** [comment]
Feel like this is too high on mobile. Maybe try this: 
image

**Comment by NathanInnes** [comment]
Please add updated screenshots

**Comment by ErickSharp** [comment]
Updated

**Review by NathanInnes — APPROVED** [review]

**Pull request merged**